app/model.php

```diff
@@ -2,7 +2,10 @@
 
 class Model
 {
-
+	private $servername = '127.0.0.1';
+	private $dbname = 'tnsdb';
+	private $username = 'root';
+	private $password = 'kilamoChi';
 
```

Comment:
sorry i forgot to tell you i moved some stuff around
can you remove these pls :)